go/tools/builders/checker_main.go

```diff
@@ -44,39 +43,54 @@ import (
 // do not return an error so as not to unnecessarily interrupt builds.
 func run(args []string) error {
 	archiveFiles := multiFlag{}
+	importMapIn := multiFlag{}
+	filenames := multiFlag{}
 	flags := flag.NewFlagSet("checker", flag.ContinueOnError)
 	flags.Var(&archiveFiles, "archivefile", "Archive file of a direct dependency")
+	flags.Var(&importMapIn, "importmap", "Import maps of a direct dependency")
```

Comment:
Thanks for the pointer. This change has not been merged into feature/check yet.

I should be able to reconstruct the information I need from the importcfg file. However, I noticed that the importcfg files uses the [host-specific GOROOT](https://github.com/bazelbuild/rules_go/blob/master/go/tools/builders/compile.go#L185) as the base path for standard library packages. My implementation uses `go.toolchain.sdk.root_file.dirname` instead. Is one of these paths preferred to the other?

Also, having the checker binary read importcfg will add some I/O overhead to the program. I'm not sure how whether this will affect performance noticeably.